yellowbrick/features/pca.py

```diff
@@ -207,6 +214,10 @@ def finalize(self, **kwargs):
         self.ax.set_ylabel('Principal Component 2')
         if self.proj_dim == 3:
             self.ax.set_zlabel('Principal Component 3')
+        if self.legend_labels is not None:
+              manual_legend(
+              self.ax, self.legend_labels, self.color, frameon=True, loc='best'
```

Comment:
We can change `self.legend_label` to `self.classes_` here. We should also be careful that `self.color` will work as expected -- have you experimented with a few different inputs for the color/colormap params? It might be easiest to explore in a Jupyter notebook. 

Also, do we actually need to pass in `self.ax`?